recipes/label_parsing.ipynb

```diff
@@ -0,0 +1,3063 @@
+{
```

Comment:
<p>"The NER in PDFs json response format is a bit complex, and thus requires to use the <code>.annotations</code> attribute a second time." -&gt; "When exported as json_response, the format of NER annotations added to PDF assets is a bit complex and requires using the <code>.annotations</code> attribute for the second time."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='119'/>